frontend/src/design/components/UserModal.js

```diff
@@ -0,0 +1,102 @@
+import React, { useState, useEffect } from 'react';
+import {
+  Typography,
+  Button,
+  Box,
+  Table,
+  TableHead,
+  TableRow,
+  TableCell,
+  TableBody,
+  Dialog,
+  CircularProgress
+} from '@mui/material';
+import { useClient } from '../../services';
+import { useDispatch } from '../../globalErrors';
+import { listUserForGroup } from '../../services/graphql/Groups/listUserForGroup';
+
+const UserModal = ({ teams, open, onClose }) => {
```

Comment:
nit: plural `teams` but we only accept a singular team/group for the API to return successfully. Think that is fine design wise but can we rename to `team` everywhere?

Reply:
Thanks, changed this everywhere.